server/controllers/finance/generalLedger/index.js

```diff
@@ -26,11 +27,44 @@ const PERIODS = [
 // expose to the API
 exports.list = list;
 exports.getAggregates = getAggregates;
+exports.getTransactions = getTransactions;
 
 // expose to server controllers
+exports.findTransactions = findTransactions;
 exports.getAccountTotalsMatrix = getAccountTotalsMatrix;
 exports.getAccountTotalsMatrixAggregates = getAccountTotalsMatrixAggregates;
 
+/**
+ * @function findTransactions
+ * @description returns general ledger transactions
+ * @param {object} options
+ */
+function findTransactions(options) {
```

Comment:
We should probably figure out what API the journal is using and just use that in the long run, but since we are short on time ... this is a good fix 👍